src/AbstractEnum.php

```diff
@@ -2,24 +2,30 @@
 
 namespace Greg0ire\Enum;
 
+use Doctrine\Common\Inflector\Inflector;
+
 /**
  * @author Grégoire Paris <postmaster@greg0ire.fr>
  * @author Sullivan Senechal <soullivaneuh@gmail.com>
  */
 abstract class AbstractEnum
 {
+    const DEFAULT_NAMESPACE_SEPARATOR = '_';
+
     private static $constCache = [];
 
     /**
      * Uses reflection to find the constants defined in the class and cache
      * them in a local property for performance, before returning them.
      *
      * @param callable|null $keysCallback
+     * @param bool          $classPrefixed      True if you want the enum class prefix on each keys, false otherwise.
+     * @param string        $namespaceSeparator Only relevant if $classPrefixed is set to true.
      *
      * @return array a hash with your constants and their value. Useful for
      *               building a choice widget
      */
-    final public static function getConstants($keysCallback = null)
+    final public static function getConstants($keysCallback = null, $classPrefixed = false, $namespaceSeparator = self::DEFAULT_NAMESPACE_SEPARATOR)
```

Comment:
No, because `short_array_syntax` is an **addition**, not a conflict.

Symfony never says we have to use long array syntax, they keep them for now to avoid conflict nightmare.

I'm pretty sure they will use long_array_syntax after 2.x maintenance support.